rclcpp/include/rclcpp/subscription_options.hpp

```diff
@@ -75,6 +75,70 @@ struct SubscriptionOptionsBase
   TopicStatisticsOptions topic_stats_options;
 
   QosOverridingOptions qos_overriding_options;
+
+  /// Enable use_default_callbacks setting.
+  SubscriptionOptionsBase &
+  enable_use_default_callbacks()
+  {
+    use_default_callbacks = true;
+    return *this;
+  }
+
+  /// Enable use_default_callbacks setting.
+  SubscriptionOptionsBase &
+  disable_use_default_callbacks()
+  {
+    use_default_callbacks = false;
+    return *this;
+  }
+
+  /// Enable ignore_local_publications setting.
+  SubscriptionOptionsBase &
+  enable_ignore_local_publications()
+  {
+    ignore_local_publications = true;
+    return *this;
+  }
+
+  /// Disable ignore_local_publications setting.
+  SubscriptionOptionsBase &
+  disable_ignore_local_publications()
+  {
+    ignore_local_publications = false;
+    return *this;
+  }
+
+  /// Set callback group, parameter idiom style.
+  SubscriptionOptionsBase &
+  set_callback_group(rclcpp::CallbackGroup::SharedPtr group)
+  {
+    callback_group = group;
+    return *this;
+  }
+
+  /// Enable intra_process_comm setting.
+  SubscriptionOptionsBase &
+  enable_use_intra_process_comm()
+  {
+    use_intra_process_comm = IntraProcessSetting::Enable;
+    return *this;
+  }
+
+  /// Disable intra_process_comm setting.
+  SubscriptionOptionsBase &
+  disable_use_intra_process_comm()
+  {
+    use_intra_process_comm = IntraProcessSetting::Disable;
+    return *this;
+  }
+
+  /// Clear intra_process_comm setting
+  SubscriptionOptionsBase &
+  clear_use_intra_process_comm()
```

Comment:
I'm not sure this is needed often enough to need its own method. This is the default state of the subscriptions options, so if you're using this function you've changed the default and you're changing it back... I find it unlikely that this scenario overlaps with one-liners, but maybe I'm missing some obvious case.

Normally, I wouldn't care if this is unlikely to be used, but if we eliminated it then we could make this `set_use_intra_process_comm(bool)` instead.

The middle ground is `set_use_intra_process_comm(IntraProcessSetting)`, but that looses some of the brevity. Though honestly, I prefer that to the current set of three methods with awkward sounding names. But it is subjective.

Reply:
(Almost same comment as above)
Thank you for comments.

It looks OK to treat it as if this is bool because one-liners are the target.

In 5ffeb9b, I think we can unify `set_some(bool)`.